demo/src/main/java/com/basgeekball/awesomevalidation/demo/DemoActivity.java

```diff
@@ -158,6 +138,13 @@ private void initValidation(String style) {
 
     private void addValidationForEditText(Activity activity) {
         mAwesomeValidation.addValidation(activity, R.id.edt_userid, "[a-zA-Z0-9_-]+", R.string.err_userid);
+        mAwesomeValidation.addValidation(activity, R.id.edt_userid, new CustomValidation() {
```

Comment:
Please remove this validation, same as above, and you have a good demonstration below anyway.

Reply:
It was like an alternative for the default validation, presenting the user with all of the options, however, removed!